### PR TITLE
add simonmicheals/hledger 1.29.2

### PR DESCRIPTION
**Pull request by RagibHasin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [ ] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/102649)

### CODE REVIEW

**Comment by RagibHasin** [comment]
@microsoft-github-policy-service agree

File: manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+LicenseUrl: https://github.com/simonmichael/hledger/blob/master/LICENSE
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Robust, fast, intuitive plain text accounting tool with CLI, TUI and web interfaces. 
+Description: hledger is lightweight, cross platform, multi-currency, double-entry accounting software. It lets you track money, investments, cryptocurrencies, invoices, time, inventory and more, in a safe, future-proof plain text data format with full version control and privacy.
+Moniker: hledger
+# Tags:
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/simonmichael/hledger/releases/tag/hledger-web-1.29.1
```
Comment (mdanish-kh) [inline manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml:24]:
```suggestion
ReleaseNotesUrl: https://github.com/simonmichael/hledger/releases/tag/1.29.2
```
ReleaseNotesUrl needs to be updated

File: manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,21 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: simonmichael.hledger
+PackageVersion: 1.29.2
+InstallerType: zip
+FileExtensions:
+- exe
+ReleaseDate: 2023-04-08
```
Comment (mdanish-kh) [inline manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.installer.yaml:9]:
```suggestion
ReleaseDate: 2023-04-07
```
According to https://hledger.org/release-notes.html#hledger-1-29-2

File: manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,29 @@
+# Created with YamlCreate.ps1 v2.2.0 $debug=NVS1.CRLF.5-1-19041-1682.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: simonmichael.hledger
+PackageVersion: 1.29.2
+PackageLocale: en-US
+Publisher: simonmichael
+PublisherUrl: https://github.com/simonmichael/hledger
+# PublisherSupportUrl:
```
Comment (mdanish-kh) [inline manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml:9]:
```suggestion
PublisherUrl: https://github.com/simonmichael/
PublisherSupportUrl: https://hledger.org/support.html
```
Update metadata

File: manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+ShortDescription: Robust, fast, intuitive plain text accounting tool with CLI, TUI and web interfaces. 
+Description: hledger is lightweight, cross platform, multi-currency, double-entry accounting software. It lets you track money, investments, cryptocurrencies, invoices, time, inventory and more, in a safe, future-proof plain text data format with full version control and privacy.
+Moniker: hledger
+# Tags:
+# Agreements:
+# ReleaseNotes:
+ReleaseNotesUrl: https://github.com/simonmichael/hledger/releases/tag/hledger-web-1.29.1
+# PurchaseUrl:
+# InstallationNotes:
+# Documentations:
```
Comment (mdanish-kh) [inline manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml:27]:
```suggestion
Documentations:
- DocumentLabel: hledger Manual
  DocumentUrl: https://hledger.org/hledger.html
- DocumentLabel: Developer docs
  DocumentUrl: https://hledger.org/dev.html
- DocumentLabel: FAQ
  DocumentUrl: https://hledger.org/faq.html
```

Taking the opportunity to suggest some more useful metadata

File: manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+PackageName: hledger
+PackageUrl: https://github.com/simonmichael/hledger
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/simonmichael/hledger/blob/master/LICENSE
+# Copyright:
+# CopyrightUrl:
+ShortDescription: Robust, fast, intuitive plain text accounting tool with CLI, TUI and web interfaces. 
+Description: hledger is lightweight, cross platform, multi-currency, double-entry accounting software. It lets you track money, investments, cryptocurrencies, invoices, time, inventory and more, in a safe, future-proof plain text data format with full version control and privacy.
+Moniker: hledger
+# Tags:
```
Comment (mdanish-kh) [inline manifests/s/simonmichael/hledger/1.29.2/simonmichael.hledger.locale.en-US.yaml:21]:
```suggestion
Tags:
- finance
- haskell
- accounting
- tui
- web-application
- ledger
- bounty
- plain-text
- command-line-tool
- console-application
- plaintext
- plaintext-accounting
- bounties
- haskell-application
- plain-text-accounting
```
Taking the opportunity to suggest some more useful metadata

**Review by mdanish-kh — CHANGES_REQUESTED** [review]

**Review by mdanish-kh — APPROVED** [review]

**Pull request merged**